doc/HOWTO_RELEASE.rst.txt

```diff
@@ -69,8 +68,8 @@ reported.
 Tool chain
 ==========
 We build all our wheels on cloud infrastructure - so this list of compilers is
-for information and debugging builds locally.  See the ``.travis.yml`` and
-``appveyor.yml`` scripts in the `numpy wheels`_ repo for the definitive source
+for information and debugging builds locally.  See the ``.travis.yml`` script
```

Comment:
I don't think  `numpy-wheels` applies anymore, especially for local builds. The manylinux wheels are built in docker images with various different compilers and the wheel builds overall are run by `multibuild`, which is hard to decipher. I don't know that anyone has made a release working on any platform besides linux, and really, a compiler isn't needed for the release except for local testing, for which a working `runtests.py` is needed. There is a list of needed packages in the `release_requirements.txt` file that can be installed with pip.